search/tavily/main.py

```diff
@@ -83,5 +75,44 @@ def main():
     print(response)
 
 
+def check_allowed_include_domains(include_domains: List[str]) -> List[str]:
+    # TAVILY_ALLOWED_DOMAINS has the TAVILY_ prefix as it will be set by Obot directly in the env,
+    # while e.g. INCLUDE_DOMAINS is a tool parameter
+    allowed_domains_str = os.getenv("TAVILY_ALLOWED_DOMAINS", "")
+    allowed_domains = [
+        domain.strip() for domain in allowed_domains_str.split(",") if domain.strip()
+    ]
+
+    if len(allowed_domains) == 0:
+        print("No allowed domains provided")
+        sys.exit(1)
+
+    # allow not setting INCLUDE_DOMAINS -  fallback to all allowed domains
+    if len(include_domains) == 0:
+        include_domains = allowed_domains
+
+    if len(allowed_domains) > 0:
```

Comment:
This is a redundant `if` statement, since we already exit if `len(allowed_domains) == 0` earlier in this function.

Reply:
Yup.. refactored without looking around